src/CSSList/CSSList.php

```diff
@@ -296,6 +296,27 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before or after its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
+     * @param string $sPosition The position.
+     *
+     * @return void
+     */
+    public function insert($oItem, $oSibling, string $sPosition = 'before'): void
```

Comment:
A comment concerning architecture:

I'd prefer to have separate methods with distinct, specific responsibilities and communicative names instead of a single method that combines multiple behaviors. The `$sPosition` behavior-switching parameter is a tell-tale sign that we should split up the method.

I suggest adopting the naming of the corresponding methods from the native `DOMElement` class: https://www.php.net/manual/en/class.domelement.php

Reply:
So the original comment by @raxbg in https://github.com/MyIntervals/PHP-CSS-Parser/issues/115#issuecomment-553319653 is no longer valid?

Would a single `insertBefore()` be ok? _(See: https://www.php.net/manual/en/domnode.insertbefore.php)_

Reply:
> So the original comment by @raxbg in https://github.com/MyIntervals/PHP-CSS-Parser/pull/115#issuecomment-553319653 is no longer valid?

Correct. We're a different bunch of people now with different opinions/preferences. :-)

Reply:
> Would a single `insertBefore()` be ok?

If that's sufficient for your usecase, yes, of course.